app/scripts/models/account.js

```diff
@@ -395,6 +395,8 @@ define(function (require, exports, module) {
         var email = this.get('email');
         var sessionToken = this.get('sessionToken');
 
+        this._metrics.logEvent('core.signin.attempt');
```

Comment:
I am open to metric naming convention. Picked `core` because I was thinking that it was a core type of metric.